modules/Router.js

```diff
@@ -15,5 +15,64 @@
    * @class  Router
    * @author kent.willis@vecna.com
    */
-  return Backbone.Router.extend({});
+
+  var Router = Backbone.Router.extend({
+
+    constructor: function(){
+      Backbone.Router.apply(this, arguments);
+      this.__applyTimingEventListeners(['pre-route', 'post-route']);
```

Comment:
@kentmw is there a better way to add all event listeners without enumerating each one as strings in the constructor?